config.yaml

```diff
@@ -47,7 +47,13 @@ services:
                     retry_delay: 250
                     retry_limit: 10
                     show_sql: false
+                  mathoid:
+                    host: '{env(MATHOID_HOST_PORT,http://localhost:10044)}'
                   purged_cache_control: s-maxage=864000, max-age=86400
+          /{domain:wikimedia.org}:
```

Comment:
Do we strictly need wikimedia.org for math, or is this more to get closer to a production like setup? For small local installs it might be a bit surprising to see wikimedia.org as a domain.